### PR TITLE
Various things

### PR DESCRIPTION
**Pull request by Tristramg**

### CODE REVIEW

File: src/stoppoints_discovery.rs [hunk 1]
```diff
@@ -58,5 +55,29 @@ pub fn stoppoints_discovery(
             }),
             ..Default::default()
         },
-    }))
+    }
+}
+
+pub fn sp_discovery(
+    (actor_addr, query): (State<Addr<Context>>, Query<Params>),
```
Comment (antoine-de) [inline src/stoppoints_discovery.rs]: by curiosity can't this be changed to `actor_addr: Addr<Context>, query: Params` too ?

Comment (Tristramg) [reply src/stoppoints_discovery.rs]: is it done somewhere else? I don’t find how to do that with actix-web

Comment (antoine-de) [reply src/stoppoints_discovery.rs]: I though so, but It's not really important

File: src/context.rs [hunk 2]
```diff
@@ -65,6 +66,14 @@ pub struct Context {
     pub data: Arc<Mutex<Data>>,
 }
 
+impl Actor for Context {
+    type Context = actix::Context<Self>;
+
+    fn started(&mut self, _ctx: &mut Self::Context) {
+        println!("Starting the context actor");
```
[the remark this answers is not in the document]
Comment (antoine-de) [reply src/context.rs]: I don't think this is resolve no ?

File: src/stop_monitoring.rs [hunk 3]
```diff
@@ -173,12 +175,9 @@ fn realtime_update(context: &Context) -> actix_web::Result<()> {
     apply_latest_rt_update(context)
 }
 
-pub fn stop_monitoring(
-    (state, query): (State<Context>, Query<Params>),
-) -> Result<Json<model::SiriResponse>> {
-    let request = query.into_inner();
+pub fn stop_monitoring(request: &Params, state: &Context) -> Result<model::SiriResponse> {
```
[the remark this answers is not in the document]
Comment (antoine-de) [reply src/stop_monitoring.rs]: Can't you remove this `pub` ?

**Comment by Tristramg** [comment]
changes for #42 have been integrated here.

**Pull request merged**